pkg/api/api.go

```diff
@@ -362,14 +362,19 @@ func (hs *HTTPServer) registerRoutes() {
 				})
 			})
 
-			if hs.ThumbService != nil {
-				dashboardRoute.Get("/uid/:uid/img/:kind/:theme", hs.ThumbService.GetImage)
+			dashboardRoute.Group("/uid/:uid", func(dashUidRoute routing.RouteRegister) {
+				if hs.Features.IsEnabled(featuremgmt.FlagPublicDashboards) {
+					dashUidRoute.Post("/public_dashboard_config", authorize(reqSignedIn, ac.EvalPermission(ac.ActionDashboardsWrite)), routing.Wrap(hs.ShareDashboard))
```

Comment:
```suggestion
					dashUidRoute.Post("/public-config", authorize(reqSignedIn, ac.EvalPermission(ac.ActionDashboardsWrite)), routing.Wrap(hs.ShareDashboard))
```

Looking at other routes we use - to separate words not underscore. And think the word dashboard here is superfluous